chat.js

```diff
@@ -953,6 +953,20 @@ Chat.escapeHTML = function (str) {
 };
 
 /**
+ * Notifies an array of rooms of a message
```

Comment:
+1 to staying in ``Rooms``.

... This part should have been its own commit and even PR too.